.header.md

```diff
@@ -287,6 +287,26 @@ Key points about VPC endpoint types:
 - **Internal endpoint**: Created when `address_allocation_ids` are omitted
 - Internal endpoints are only accessible from within the VPC or connected networks
 
+## Example for Public Endpoint Configuration
+
+This example demonstrates a simple public endpoint configuration:
+
+```hcl
+module "transfer_server" {
```

Comment:
could you give an example of a configuration that is needed to leverage multiple keys? If you can add exactly what is needed for customers to use multiple keys